db/migrate/20160818091421_add_missing_view_permissions2.rb

```diff
@@ -0,0 +1,21 @@
+class AddMissingViewPermissions2 < ActiveRecord::Migration
+  def up
+    default_permissions = Foreman::Plugin.find("foreman_discovery").default_roles
+    ["Discovery Reader", "Discovery Manager"].each do |role_name|
+      role = Role.find_by_name(role_name) || next
+      puts "Processing #{role}"
+      default_permissions[role_name].each do |permission|
+        unless role.permission_names.include?(permission.to_sym)
+          puts "Adding permission #{permission}"
```

Comment:
do we want to actually show it when the migration is running or just add it to logs?